test/mobile/test_bytecode.py

```diff
@@ -15,33 +25,231 @@
 # optimized_scripted_module = optimize_for_mobile(script_module)
 # exported_optimized_scripted_module = optimized_scripted_module._save_for_lite_interpreter(
 #   str(output_model_path))
 
+SCRIPT_MODULE_V4_path_BYTECODE_PKL = '''
+(4,
+ ('__torch__.*.TestModule.forward',
+  (('instructions',
+    (('STOREN', 1, 2),
+     ('DROPR', 1, 0),
+     ('LOADC', 0, 0),
+     ('LOADC', 1, 0),
+     ('MOVE', 2, 0),
+     ('OP', 0, 0),
+     ('LOADC', 1, 0),
+     ('OP', 1, 0),
+     ('RET', 0, 0))),
+   ('operators', (('aten::add', 'int'), ('aten::add', 'Scalar'))),
+   ('constants',
+    (torch._utils._rebuild_tensor_v2(pers.obj(('storage', torch.DoubleStorage, '0', 'cpu', 8),),
+       0,
+       (2, 4),
+       (4, 1),
+       False,
+       collections.OrderedDict()),
+     1)),
+   ('types', ()),
+   ('register_size', 2)),
+  (('arguments',
+    ((('name', 'self'),
+      ('type', '__torch__.*.TestModule'),
+      ('default_value', None)),
+     (('name', 'y'), ('type', 'int'), ('default_value', None)))),
+   ('returns',
+    ((('name', ''), ('type', 'Tensor'), ('default_value', None)),)))))
+        '''
+
+SCRIPT_MODULE_V5_path_BYTECODE_PKL = '''
+(5,
+ ('__torch__.*.TestModule.forward',
+  (('instructions',
+    (('STOREN', 1, 2),
+     ('DROPR', 1, 0),
+     ('LOADC', 0, 0),
+     ('LOADC', 1, 0),
+     ('MOVE', 2, 0),
+     ('OP', 0, 0),
+     ('LOADC', 1, 0),
+     ('OP', 1, 0),
+     ('RET', 0, 0))),
+   ('operators', (('aten::add', 'int'), ('aten::add', 'Scalar'))),
+   ('constants',
+    (torch._utils._rebuild_tensor_v2(pers.obj(('storage', torch.DoubleStorage, 'constants/0', 'cpu', 8),),
+       0,
+       (2, 4),
+       (4, 1),
+       False,
+       collections.OrderedDict()),
+     1)),
+   ('types', ()),
+   ('register_size', 2)),
+  (('arguments',
+    ((('name', 'self'),
+      ('type', '__torch__.*.TestModule'),
+      ('default_value', None)),
+     (('name', 'y'), ('type', 'int'), ('default_value', None)))),
+   ('returns',
+    ((('name', ''), ('type', 'Tensor'), ('default_value', None)),)))))
+        '''
+
+SCRIPT_MODULE_BYTECODE_PKL = {
+    4: {
+        "bytecode_pkl": SCRIPT_MODULE_V4_path_BYTECODE_PKL,
+        "model_name": "script_module_v4.ptl"},
+    5: {
+        "bytecode_pkl": SCRIPT_MODULE_V5_path_BYTECODE_PKL,
+        "model_name": "script_module_v5.ptl"},
+}
+
+MINIMUM_TO_VERSION = 4
+
 class testVariousModelVersions(TestCase):
     def test_get_model_bytecode_version(self):
-        script_module_v4 = pytorch_test_dri / "cpp" / "jit" / "script_module_v4.ptl"
-        version_v4 = _get_model_bytecode_version(script_module_v4)
+        script_module_v4_path = pytorch_test_dri / "cpp" / "jit" / "script_module_v4.ptl"
+        script_module_v5_path = pytorch_test_dri / "cpp" / "jit" / "script_module_v5.ptl"
+
+        version_v4 = _get_model_bytecode_version(script_module_v4_path)
+        version_v5 = _get_model_bytecode_version(script_module_v5_path)
+
         assert(version_v4 == 4)
```

Comment:
I think this is missing the assertion to v5
Plus now you can write this with a loop using SCRIPT_MODULE_BYTECODE_PKL

Reply:
Resolve with for loop